test/Google.Api.Gax.Grpc.Tests/RetryTest.cs

```diff
@@ -96,30 +98,38 @@ public class RetryTest
             await scheduler.RunAsync(async () =>
             {
                 var callSettings = CallSettings.FromCallTiming(CallTiming.FromRetry(retrySettings));
-                var retryingCallable = callable.WithRetry(scheduler.Clock, scheduler);
-                var result = await Call(async, retryingCallable, new SimpleRequest { Name = name }, callSettings);
+                SimpleResponse result;
```

Comment:
Given that we've got this if/else several times, perhaps move it to a common method?

Reply:
OK, yes, I'll see about doing a small refactor.